fake_switches/brocade/command_processor/config_virtual_interface_vrrp.py

```diff
@@ -16,12 +16,9 @@
 
 
 class ConfigVirtualInterfaceVrrpCommandProcessor(ConfigInterfaceCommandProcessor):
-    def __init__(self, switch_configuration, terminal_controller, logger,
-                 piping_processor, port, vrrp):
-        super(ConfigVirtualInterfaceVrrpCommandProcessor, self).__init__(
-            switch_configuration, terminal_controller, logger, piping_processor,
-            port)
-        self.vrrp = vrrp
+    def init(self, switch_configuration, terminal_controller, logger, piping_processor, *args):
+        super(ConfigVirtualInterfaceVrrpCommandProcessor, self).init(switch_configuration, terminal_controller, logger, piping_processor, args[0])
```

Comment:
what is args[0] here ?
For me it's unclear.

I suggest to take a better contract than *args. My suggestion is **kwargs but not sure if it's the best.
Another solution would be to name this parameter prior passing it to ConfigVirtualInterfaceVrrpCommandProcessor